lib/screens/alerts/alert.ex

```diff
@@ -377,4 +384,36 @@ defmodule Screens.Alerts.Alert do
   def informs_stop_id?(%__MODULE__{informed_entities: informed_entities}, stop_id) do
     Enum.any?(informed_entities, &(&1.stop == stop_id))
   end
+
+  @spec normalize_informed_entities_for_direction_id([Alert.t()]) :: [Alert.t()]
+  def normalize_informed_entities_for_direction_id(alerts) do
+    Enum.map(alerts, fn alert ->
+      %{
```

Comment:
Being explicit here and using `%__MODULE__{` gets us better compile-time checks and LSP functionality.